### PR TITLE
[FEATURE] Add setting to increase the font size of the name under a character on the canvas

### LINKED ISSUE #691: Add setting to increase the font size of the name under a character on the canvas
We already allow increase the chat font size in the setting, but there you should also be able to increase the font size of the name under a character on the canvas

### PR DESCRIPTION
**Pull request by Angela-BC**
closes #691 

Adds a font size select drop down to settings - interface - chatroom UI.
Renders the name on the canvas with different sizes depending on the selected font size.

### CODE REVIEW

File: pandora-client-web/src/graphics/room/roomCharacter.tsx
Comment (ClaudiaMia) [inline pandora-client-web/src/graphics/room/roomCharacter.tsx:306]: I feel it might still not be big enough for some people.

Maybe we should actually increase the default a bit from the start, considering many may have smaller screens?

Then the factors could be more like: 0.6 / 1.0 / 1.4 (new default) /  1.8 / 2.2

**Review by ClaudiaMia — APPROVED** [review]
Aside from my optional comment, it looks good to me and I tested that it works well.

Needs second review.

File: pandora-client-web/src/graphics/room/roomCharacter.tsx
Comment (Angela-BC) [reply pandora-client-web/src/graphics/room/roomCharacter.tsx:306]: Committed a new version with the proposed values. 
On small screens it is still not very large.

**Review by Sekkmer — APPROVED** [review]
Looks good to me

(NOTE part is good enough, we can refine it later if needed)

**Pull request merged**
